src/utils.js

```diff
@@ -0,0 +1,21 @@
+const dateFormat = require('dateformat');
+
+const timestamp = () => dateFormat(new Date(), 'yyyy-mm-dd.HHMMss');
+
+const parseArray = stringArray => {
```

Comment:
nit: I'd prefer naming like, "stringArray" makes me think it's type `string[]`:

 `const getArgumentsFromCommaDelimitedString = (commaDelimitedString)` ?

Reply:
I definitely was not in love with that naming convention - thanks for bringing it up!